### PR TITLE
RLS: Add release notes for release 1.3.0

### LINKED ISSUE #2028: RLS: 1.3
we should do a release soon, as last was in mid-summer.

anyone interested in being release manager? also an opportunity to update docs on how-to-release.

### PR DESCRIPTION
**Pull request by xmnlab**
Resolves #2028

### CODE REVIEW

File: docs/source/release.rst [hunk 1]
```diff
@@ -7,33 +7,109 @@ Release Notes
...
+* :bug:`1883 major` Fix small formatting buglet in PR merge tool
+* :support:`1882` Added CoC section
+* :bug:`1876 major` Fix the case where we do not have an index when using preceding with intervals
+* :feature:`1807` Spark client
+* :bug:`1872 major` Fixed issues with geo data
+* :feature:`1868` Use pandas rolling apply to implement rows_with_max_lookback
+* :bug:`1869 major` Remove -x from pytest call in linux CI
+* :bug:`1867 major` Fix return type of Struct.from_tuples
+* :support:`1859` Remove pandas exception for rows_with_max_lookback
+* :support:`1856` Move CI pipelines to Azure
```
Comment (jreback) [inline docs/source/release.rst]: instead of jamming all of the notes together
can i make subsections. for each release 
eg 1.2 1.3 etc

Comment (xmnlab) [reply docs/source/release.rst]: do you mean one file for each release note like pandas? if yes, do you have an specific file structure in mind?

Comment (jreback) [reply docs/source/release.rst]: no you just add something like

```
Version 1.3
------------
```

before each section

pls build the docs and you will see

Comment (xmnlab) [reply docs/source/release.rst]: oh I got it. thanks, I will work on that.

**Comment by xmnlab** [comment]
it seems sphinx-release automatically create these sub sections (https://docs.ibis-project.org/release.html)

it seems the manual sub sections break sphinx-release.

**Comment by jreback** [comment]
> it seems sphinx-release automatically create these sub sections (https://docs.ibis-project.org/release.html)
> 
> it seems the manual sub sections break sphinx-release.

interesting. ok then the existing automatation is ok.

File: docs/source/release.rst [hunk 2]
```diff
@@ -7,33 +7,109 @@ Release Notes
...
+* :feature:`1860` Add shapely geometries as input for literals
+* :bug:`1910 major` Fix union implementation
+* :bug:`1912 major` Fix failing com imports on master
+* :feature:`1858` Add geopandas as output for omniscidb
+* :bug:`1901 major` OmniSci/MapD - Fix reduction for bool
+* :feature:`1885` Spark UDFs
+* :feature:`1871` Add support for Postgres UDFs
+* :bug:`1899 major` Pass scope to grouping execution in the pandas backend
+* :support:`1895` Update link to Presto website
+* :support:`1896` Removing linting from windows
```
Comment (jreback) [inline docs/source/release.rst]: there are no duplicates here?

Comment (xmnlab) [reply docs/source/release.rst]: I check it now and I didn't find any duplicates there.

**Comment by jreback** [comment]
great thanks @xmnlab

**Pull request merged**